edivorce/apps/core/templates/question/06_children_income_expenses.html

```diff
@@ -234,13 +234,13 @@ <h3>What is the monthly child support amount (as per
                         <td>use the table for that province or territory.</td>
                     </tr>
                     <tr>
-                        <td>you live in different provinces or territories and one of you has sole custody and the other
+                        <td>you live in different provinces or territories and one of you has a sole parenting order and the other
                             parent must pay support
                         </td>
                         <td>use the table for the province or territory where the paying parent lives.</td>
                     </tr>
                     <tr>
-                        <td>you live in different provinces or territories and you share or split custody</td>
+                        <td>you live in different provinces or territories and you share or a split parenting order</td>
```

Comment:
The grammar doesn't seem right 

Should it be "share or split a parenting order"?